docs/modules/third_party_rules_callbacks.md

```diff
@@ -247,6 +247,24 @@ admin API.
 
 If multiple modules implement this callback, Synapse runs them all in order.
 
+### `on_threepid_bind`
+
+_First introduced in Synapse v1.56.0_
+
+```python
+async def on_threepid_bind(user_id: str, medium: str, address: str) -> None:
+```
+
+Called after creating an association between a local user and a third-party identifier
```

Comment:
Maybe just clarify that the association here is on the homeserver?

Reply:
It does say below:

> Note that this callback is _not_ called after a successful association on an _identity
> server_.

Which I think is enough?

Reply:
I saw that and my only thought was that the below statement makes one draw an inference rather than explicitly stating that the association in question is on the homeserver but it's a very subtle point!

Reply:
I see what you mean. Considering the only alternative to the association happening on the IS is that it's happening on the HS, I think that should be good enough. We can always revisit it later if we find that it confuses people though.